tensorflow_addons/metrics/tests/harmonic_mean_test.py

```diff
@@ -0,0 +1,116 @@
+# Copyright 2020 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Tests HarmonicMean metrics."""
+
+import numpy as np
+import pytest
+import tensorflow as tf
+
+from tensorflow_addons.metrics import HarmonicMean
+
+
+def get_test_data():
```

Comment:
As the base class is covered by https://github.com/tensorflow/tensorflow/blob/v2.4.0/tensorflow/python/keras/metrics_test.py#L169 what we really need to test here for the overrided methods?

Reply:
Okay, I'll remove all the extra tests (config tests, state tests, etc.)